templates/todo/projects/nodejs-mongo-aks/src/api/manifests/deployment.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: todo-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: todo-api
+  template:
+    metadata:
+      labels:
+        app: todo-api
+    spec:
+      containers:
+        - name: todo-api
+          image: crppnnadqdq7owqmxjqvwcid425o.azurecr.io/todo-nodejs-mongo-aks/api:latest
+          ports:
+            - containerPort: 3100
+          env:
+            - name: AZURE_CLIENT_ID
+              value: 8111defe-2f69-4893-b025-ffeaea359c4a
```

Comment:
Guessing this is another place we'll have to do replacements?  I assume this client ID is specific to something about the identity assigned inside the cluster.